content/java/concepts/strings/terms/subSequence/subSequence.md

```diff
@@ -0,0 +1,48 @@
+---
+Title: '.subSequence()'
+Description: 'Returns a character sequence from a string.'
+Subjects:
+  - 'Computer Science'
+Tags:
+  - 'Strings'
+  - 'Methods'
+CatalogContent:
+  - 'learn-java'
+  - 'paths/computer-science'
+---
+
+**`.subSquence()`** is a built-in method that returns a new character sequence that is a subsequence of a user-specified string.
+
+## Syntax
+
+```pseudo
+str.subSequence(int startIndex, int endIndex)
+```
+
+The `subSequence()` method will return a subsequence of the `str` string that starts with the character at the position `startIndex` and ends with the character at the position `endIndex -1`
+
+The `.subSquence()` method takes two parameters:
+  - `startIndex` - the starting index (inclusive);
+  - `endIndex` - the ending index (exclusive).
+
+  Both parameters must be `integers`.
+
+## Example
+
+The following example will return a subsequence of the `str` string.
+
+```java
```

Comment:
The example code should be corrected in the following way:
1. Use two spaces as opposed to tabs for indention.
2. The method is `.subSequence()` not `.subsequence()`.
3. `.printlm()` is a typo (line 39)! it should be `System.out.println(str.subsequence(0, 4))`.